src/OpenSearch.Net/README.md

```diff
@@ -0,0 +1,3 @@
+A low-level, dependency free OpenSearch .Net client.
+
+It provides a one-to-one mapping with the OpenSearch REST API. The client is completely matches the official REST API specification and supports all OpenSearch features. The client also has support for load balancing and cluster failover and all client method calls have both synchronous and asynchronous variants
```

Comment:
Can remove `is` and `completely`:
`The client matches the official REST API specification and supports all OpenSearch features.`